libs/api-client-bear/src/gooddata-browser.ts

```diff
@@ -1,14 +1,16 @@
-// (C) 2007-2020 GoodData Corporation
+// (C) 2007-2022 GoodData Corporation
 import "isomorphic-fetch";
 import { factory, SDK } from "./gooddata";
 
+const getFactoryBrowser = (config?: any) => factory(fetch.bind(window))(config);
+
 const factoryBrowser = factory(fetch.bind(window));
```

Comment:
Please mark this as deprecated and advise users to use the getFactory function (same in the node version of this file). This will help us remove the old one in SDK9.